packages/web/src/components/notification/Notifications/TwitterShareButton.tsx

```diff
@@ -8,9 +8,17 @@ const messages = {
   share: 'Share'
 }
 
-export const TwitterShareButton = () => {
+type TwitterShareButtonProps = {
+  onClick?: MouseEventHandler
+}
+
+// TODO: mark notificationAsRead() // potentially move this to tile?
```

Comment:
Ooh interesting, does this steal the click event from the parent so the notif doesn't get marked as read?

Reply:
ahh i should update this comment, i think basically we can save a few lines if we put it in the tile since clicking on the tile in any capacity should prob indicate the notif is read